Makefile

```diff
@@ -1,30 +1,21 @@
 # Version tag for all images
 TEST_INFRA_VERSION ?= "latest"
-
 # Version of the gRPC driver
 DRIVER_VERSION ?= "master"
-
 # Prefix for all images used as clone and ready containers, enabling use with registries
 # other than DockerHub
 INIT_IMAGE_PREFIX ?= ""
-
 # Prefix for all images used as build containers, enabling use with registries
 # other than DockerHub
 BUILD_IMAGE_PREFIX ?= ""
-
 # Prefix for all images used as runtime containers, enabling use with registries
 # other than DockerHub
 IMAGE_PREFIX ?= ""
-
 # Image URL to use all building/pushing image targets
 CONTROLLER_IMG ?= ${IMAGE_PREFIX}controller:${TEST_INFRA_VERSION}
-
 # Image URL to use all building/pushing image targets
 CLEAN_IMG ?= ${IMAGE_PREFIX}cleanup:${TEST_INFRA_VERSION}
-
-# Node pool for system components (controller and cleanup agent)
-SYSTEM_POOL ?= "system"
-
+#${IMAGE_PREFIX}cleanup_agent:${TEST_INFRA_VERSION}
```

Comment:
Is this comment intended here? Other than that, LGTM.

Reply:
It was in the previous version on master, so when I reverted... it came back. I can add a commit to remove it